### PR TITLE
Heroku appropriately deployed

### LINKED ISSUE #2: Deployed to Heroku

### PR DESCRIPTION
**Pull request by Malllll12**
## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [] The code will run locally

## Type of change

    [] New feature
    [x] Bug Fix
    [] Refactor

## Description of Change:

    description: Some spiked fiddling to get heroku to work

## Requested feedback:

    I'd like feedback on...

## Check the correct boxes

    [] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

**Comment by Malllll12** [comment]
Rubocop still hates me

**Review by ParkerLockhart — APPROVED** [review]
Lol Rubocop is so grumpy. 
Awesome work on the troubleshooting!! Yay!!!

**Review by dkulback — APPROVED** [review]

File: db/seeds.rb [hunk 1]
```diff
@@ -5,5 +5,9 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-big_foot = Cryptid.create(name: "Big Foot", range: "range 1", danger_level: "6", image: "Image 1", description: "description 1")
+big_foot = Cryptid.create(name: "Big Foot",
+                          range: "Rocky Mountains, Appalachian Trial, Coastal Plain",
+                          danger_level: "6",
+                          image: "https://m.media-amazon.com/images/M/MV5BZTdlMzI2MGYtMTZkNi00OWUyLTk2YWEtNTdmYzRiNjlhM2QyXkEyXkFqcGdeQXVyMjU0MzA2NjM@._V1_.jpg",
+                          description: "Sasquatch, also called Bigfoot, (from Salish se’sxac: “wild men”) a large, hairy, humanlike creature believed by some people to exist in the northwestern United States and western Canada. It seems to represent the North American counterpart of the Himalayan region’s mythical monster, the Abominable Snowman, or Yeti.")
```
Comment (echon006) [inline db/seeds.rb:12]: Awesome details

**Review by echon006 — APPROVED** [review]
everything looks good!

**Pull request merged**